data/sql/updates/pending_db_world/elder_gossip_fix.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6815, 2, 0, 'Where is Elder Moonwarden?', 11580, 1, 1, 21107, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6815, 3, 0, 'Where is Elder Windtotem?', 11576, 1, 1, 21147, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6815, 4, 0, 'Where is Elder Darkhorn?', 11584, 1, 1, 6896, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 0, 0, 'Where is Elder Darkcore?', 11493, 1, 1, 21086, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 1, 0, 'Where is Elder Ironband?', 11507, 1, 1, 6854, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 2, 0, 'Where is Elder Wheathoof?', 11582, 1, 1, 6895, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 3, 0, 'Where is Elder Runetotem?', 11588, 1, 1, 21122, 0, 0, 0, '', 0, 0);
+INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES (6899, 4, 0, 'Where is Elder Windtotem?', 11576, 1, 1, 21147, 0, 0, 0, '', 0, 0);

```

Comment:
```suggestion
DELETE FROM `gossip_menu_option` WHERE `MenuID` IN (6815, 6899);
INSERT INTO `gossip_menu_option` (`MenuID`, `OptionID`, `OptionIcon`, `OptionText`, `OptionBroadcastTextID`, `OptionType`, `OptionNpcFlag`, `ActionMenuID`, `ActionPoiID`, `BoxCoded`, `BoxMoney`, `BoxText`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES
(6815, 1, 0, 'Where is Elder High Mountain?', 11578, 1, 1, 21097, 0, 0, 0, '', 0, 0),
(6815, 2, 0, 'Where is Elder Moonwarden?', 11580, 1, 1, 21107, 0, 0, 0, '', 0, 0),
(6815, 3, 0, 'Where is Elder Windtotem?', 11576, 1, 1, 21147, 0, 0, 0, '', 0, 0),
(6815, 4, 0, 'Where is Elder Darkhorn?', 11584, 1, 1, 6896, 0, 0, 0, '', 0, 0),
(6899, 0, 0, 'Where is Elder Darkcore?', 11493, 1, 1, 21086, 0, 0, 0, '', 0, 0),
(6899, 1, 0, 'Where is Elder Ironband?', 11507, 1, 1, 6854, 0, 0, 0, '', 0, 0),
(6899, 2, 0, 'Where is Elder Wheathoof?', 11582, 1, 1, 6895, 0, 0, 0, '', 0, 0),
(6899, 3, 0, 'Where is Elder Runetotem?', 11588, 1, 1, 21122, 0, 0, 0, '', 0, 0),
(6899, 4, 0, 'Where is Elder Windtotem?', 11576, 1, 1, 21147, 0, 0, 0, '', 0, 0);
```

Reply:
I had that 2 commits ago with the exception of not combining the 2 deletes, let's see if she works...